<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Asra" failures="1" tests="5" name="testCases.TC04_CompletePurchase" time="328.418" errors="0" timestamp="2025-02-19T01:25:21 EST" skipped="2">
  <testcase classname="testCases.TC04_CompletePurchase" name="testCompletePurchase" time="72.200"/>
  <testcase classname="testCases.TC04_CompletePurchase" name="testCompletePurchase" time="70.813"/>
  <testcase classname="testCases.TC04_CompletePurchase" name="testCompletePurchase" time="61.243">
    <failure message="Test encountered an unexpected error: element not interactable
  (Session info: MicrosoftEdge=133.0.3065.59)
Build info: version: &#039;4.28.1&#039;, revision: &#039;73f5ad48a2&#039;
System info: os.name: &#039;Windows 11&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;17.0.10&#039;
Driver info: org.openqa.selenium.edge.EdgeDriver
Command: [bbb92badcfaf31884f9642d56b9ba89a, clickElement {id=f.4BFA18836805DF6E899D3D9DE19A4249.d.5B3F3DE372EC51012D57B80AA182A0B5.e.144}]
Capabilities {acceptInsecureCerts: false, browserName: MicrosoftEdge, browserVersion: 133.0.3065.59, fedcm:accounts: true, ms:edgeOptions: {debuggerAddress: localhost:52639}, msedge: {msedgedriverVersion: 133.0.3065.69 (c3f010c11f3e..., userDataDir: C:\Users\asran\AppData\Loca...}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:52639/devtoo..., se:cdpVersion: 133.0.3065.59, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Element: [[EdgeDriver: MicrosoftEdge on windows (bbb92badcfaf31884f9642d56b9ba89a)] -&gt; xpath: //a[normalize-space()=&#039;HP LP3065&#039;]]
Session ID: bbb92badcfaf31884f9642d56b9ba89a" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Test encountered an unexpected error: element not interactable
  (Session info: MicrosoftEdge=133.0.3065.59)
Build info: version: '4.28.1', revision: '73f5ad48a2'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.10'
Driver info: org.openqa.selenium.edge.EdgeDriver
Command: [bbb92badcfaf31884f9642d56b9ba89a, clickElement {id=f.4BFA18836805DF6E899D3D9DE19A4249.d.5B3F3DE372EC51012D57B80AA182A0B5.e.144}]
Capabilities {acceptInsecureCerts: false, browserName: MicrosoftEdge, browserVersion: 133.0.3065.59, fedcm:accounts: true, ms:edgeOptions: {debuggerAddress: localhost:52639}, msedge: {msedgedriverVersion: 133.0.3065.69 (c3f010c11f3e..., userDataDir: C:\Users\asran\AppData\Loca...}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:52639/devtoo..., se:cdpVersion: 133.0.3065.59, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Element: [[EdgeDriver: MicrosoftEdge on windows (bbb92badcfaf31884f9642d56b9ba89a)] -> xpath: //a[normalize-space()='HP LP3065']]
Session ID: bbb92badcfaf31884f9642d56b9ba89a
at org.testng.Assert.fail(Assert.java:111)
at testCases.TC04_CompletePurchase.testCompletePurchase(TC04_CompletePurchase.java:72)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:269)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:76)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:840)
]]>
    </failure>
  </testcase> <!-- testCompletePurchase -->
  <testcase classname="testCases.TC04_CompletePurchase" name="testCompletePurchase" time="61.219">
    <skipped/>
  </testcase> <!-- testCompletePurchase -->
  <testcase classname="testCases.TC04_CompletePurchase" name="testCompletePurchase" time="62.943">
    <skipped/>
  </testcase> <!-- testCompletePurchase -->
</testsuite> <!-- testCases.TC04_CompletePurchase -->
